src/main/java/org/osiam/auth/configuration/LdapConfiguration.java

```diff
@@ -26,25 +26,29 @@
 import java.util.HashMap;
 import java.util.Map;
 
-import javax.inject.Inject;
-
 import org.osiam.auth.exception.LdapConfigurationException;
 import org.osiam.auth.login.ldap.OsiamLdapAuthenticationProvider;
 import org.osiam.auth.login.ldap.OsiamLdapUserContextMapper;
 import org.osiam.resources.scim.User;
 import org.springframework.beans.factory.InitializingBean;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.ldap.core.DirContextAdapter;
 import org.springframework.ldap.core.DirContextOperations;
-import org.springframework.security.authentication.ProviderManager;
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
+import org.springframework.security.config.annotation.method.configuration.EnableGlobalMethodSecurity;
+import org.springframework.security.config.annotation.method.configuration.GlobalMethodSecurityConfiguration;
 import org.springframework.security.ldap.DefaultSpringSecurityContextSource;
 import org.springframework.security.ldap.authentication.BindAuthenticator;
 import org.springframework.security.ldap.userdetails.DefaultLdapAuthoritiesPopulator;
 
 @Configuration
-public class LdapConfiguration implements InitializingBean {
+@EnableGlobalMethodSecurity
+@ConditionalOnProperty(prefix = "org.osiam.auth-server", name = "ldap.enabled")
+public class LdapConfiguration extends GlobalMethodSecurityConfiguration implements InitializingBean {
```

Comment:
Also, here's the tutorial you mentioned: https://spring.io/guides/gs/authenticating-ldap/